packages/odyssey-react-mui/src/PasswordField.tsx

```diff
@@ -130,6 +130,7 @@ const PasswordField = forwardRef<HTMLInputElement, PasswordFieldProps>(
             </InputAdornment>
           }
           id={id}
+          name={id}
```

Comment:
Should `name` always = `id`?

Reply:
For now, I think this is fine. If we find a use case where that doesn't make sense, we can optionally add a `name` prop as well and do `id || name`.